src/app/pages/BlockDetailPage/index.tsx

```diff
@@ -89,8 +89,12 @@ export const BlockDetailView: FC<{
       <dd>
         <BlockLink scope={block} height={block.round} />
         <CopyToClipboard value={block.round.toString()} />
-        <PrevBlockButton scope={block} currentRound={block.round} />
-        <NextBlockButton scope={block} currentRound={block.round} />
+        {!standalone && (
```

Comment:
Issue: https://5603e983.oasis-explorer.pages.dev/search?q=2%2C265%2C700
still appears on search results too

Reply:
(sidenote, i'm confused what standalone means.)

Reply:
I would expect standalone to mean single block view, not the opposite https://5603e983.oasis-explorer.pages.dev/mainnet/sapphire/block/2265700

Reply:
hard to say what it means, but I think naming does not matter now. Let's fix buttons and imo let's refactor vertical tables to rely only on css. This way we will get rid of that prop (unless it is used in more contexts that I am  not aware of)

Reply:
added `enableBlockNavigation` prop